recipes/magpylib/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "magpylib" %}
+{% set version = "1.0.2b0" %}
+{% set pypiSourceChecksum = 'fde2f8641a265461a9eb50259c5fccdf82822b57a441e3eae7a99f2861f3022e' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ pypiSourceChecksum }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python>=3.6
+    - pip
+  run:
+    - python
+    - matplotlib>=3.1
+    - numpy>=1.16
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: http://github.com/magpylib/magpylib
+  license: AGPL-3.0
```

According to the `setup.py`:
```suggestion
  license: AGPL-3.0-or-later
```